community/hazelcast-jet-enterprise/templates/statefulset.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "hazelcast-jet.fullname" . }}
+  labels:
+    app: {{ template "hazelcast-jet.name" . }}
+    chart: {{ template "hazelcast-jet.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: {{ template "hazelcast-jet.fullname" . }}
+  replicas: {{ .Values.cluster.memberCount }}
+  selector:
+    matchLabels:
+      app: {{ template "hazelcast-jet.name" . }}
+      chart: {{ template "hazelcast-jet.chart" . }}
```

Comment:
Using versions in a deployment's label selector is discouraged since they will be immutable after getting created and helm upgrade will break if there is a version change.  See https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#label-selector-updates for more details.

Reply:
Version excluded from the template